emojivoto-web/cmd/server.go

```diff
@@ -30,7 +31,7 @@ func main() {
 	}
 
 	// get TLS config
-	tlsCfg, err := marble.GetClientTLSConfig()
+	tlsCfg, err := marble.GetServerTLSConfig()
```

Comment:
This config has RequireAndVerifyClientCert set. This hasn't been the case before the last PR. Is this ok?

Reply:
Yes, that's fine because we're using a separate certificate for the webserver now.